src/components/Settings/AddPreferences.tsx

```diff
@@ -45,14 +47,26 @@ export function AddPreferences({
 
   function handleSubmission(e: any) {
     e.preventDefault();
-    setPreferenceMutation.mutate({
-      id: preference.id,
-      name: preference.name,
-      value: preference.value,
-      comment: preference.comment,
-      type: preference.type,
-      locale: t.locale,
-    });
+    const maxPreferences = isRepresentative
```

Comment:
This should also take the extra orders into account

Reply:
Do we really want the user to be able to increase the amount of preferences on their own? Might be better if we do it manually after confirming with the sales team.

Reply:
Well I was thinking that it would be best to just put in a disclaimer that all places are not yet guaranteed but that they can add all food preferences for if they get the places.

But then I guess we would need to later know which food preferences they want to keep if they don't get all the spots they want and then what was the point of getting all the food preferences in the first place?

In other words: alright, idk then :sweat_smile:

Reply:
Hmm, okay, let's leave it like this for now and discuss it with the other teams later.